packages/flutter_test/lib/src/controller.dart

```diff
@@ -252,9 +253,12 @@ abstract class WidgetController {
   /// the given widget, assuming it is exposed.
   ///
   /// If the center of the widget is not exposed, this might send events to
-  /// another object.
-  Future<void> tap(Finder finder, {int? pointer, int buttons = kPrimaryButton}) {
-    return tapAt(getCenter(finder), pointer: pointer, buttons: buttons);
+  /// another object. If `warnIfMissed` is true (the default), then a warning
```

Comment:
I find the explanation given on hitTestWarningShouldBeFatal better than this one here (especially, the example given there of what a miss could mean is helpful). Unfortunately, most people will probably read this doc over hitTestWarningShouldBeFatal. Maybe move some of that stuff into these docs?